services/file_get_contents.proto

```diff
@@ -31,47 +39,63 @@ import "query_header.proto";
 import "response_header.proto";
 
 /**
- * Get the contents of a file. The content field is empty (no bytes) if the file is empty.
+ * A query request to the Hedera File Service (HFS) for file content.
  */
```

Comment:
Add new paragraphs:
* This query requests the _content_ of a file, but none of the
* information _about_ a file. A client should submit a `fileGetInfo` query to
 * view information about a file. 
 * <br/>
 * File content _may_ also be available from a
 * block node or mirror node, generally at lower cost.